File: pkgs/lua/hello.lua
--!/opt/bin/lua
print("Hello from Lua!")

function factorial(n)
    local result = 1
    for i = 2, n do
        result = result * i
    end
    return result
end

for i = 1, 10 do
    print("factorial(" .. i .. ") = " .. factorial(i))
end

File: pkgs/mujs/hello.js
//!/opt/bin/mujs
console.log("Hello from MuJS!");

function factorial(n) {
    var result = 1;
    for (var i = 2; i <= n; i++) {
        result = result * i;
    }
    return result;
}

for (var i = 1; i <= 10; i++) {
    console.log("factorial(" + i + ") = " + factorial(i));
}